frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1067,9 +1070,15 @@ class TableInteractive extends Component {
 
     const headerHeight = this.props.tableHeaderHeight || HEADER_HEIGHT;
     const gutterColumn = this.state.showDetailShortcut ? 1 : 0;
-    const shortcutColumn = Number(!question?.isArchived());
     const query = question?.query();
     const info = query && Lib.queryDisplayInfo(query);
+    const hasAggregation = cols.some(column => column.source === "aggregation");
+
+    const shortcutColumn =
+      info?.isEditable &&
+      Number(!question?.isArchived()) &&
+      !hasAggregation &&
```

Comment:
This seems a bit hacky, but `Lib.expressionableColumns` returns a bunch of columns for the query.

Applying an expression to those columns results in a broken query.

Reply:
@bshepherdson Should this be allow at all?

Reply:
See my comments on Slack for more.

I think the right thing here isn't blocking this button, but making it work fine for summarized queries.